components/infobox/wikis/starcraft2/infobox_league_custom.lua

```diff
@@ -404,13 +406,16 @@ function CustomLeague:_cleanPrizeValue(value, currency, oldHasPlus, oldHasText)
 	--remove currency abbreviations
 	value = value:gsub('<abbr.*abbr>', ''):gsub(',', '')
 
-	--remove currency symbol
+	--remove currency symbol and text
 	if currency then
+		value = value:gsub(currency:lower(), '')
+		value = value:gsub(currency:upper(), '')
```

Comment:
Shouldn't removing currency abbr remove the currency name as well?

Reply:
no, there are events that have entries like `$ 1500 USD` wile others have `$ 1500 {{abbr/USD}}`

but i am currently bot removing all that shit, so will open a new pr to kick some of the `gsub` calls after that
(probably only the `,` and the `+` stuff will stay)

Reply:
undid the changes to function `_cleanPrizeValue`, will open a new PR for that function in a bit (since i cleaned lots of stuff via bot we can trim the function down a bit now)